uvicorn/workers.py

```diff
@@ -75,6 +76,11 @@ def run(self) -> None:
         server = Server(config=self.config)
         loop = asyncio.get_event_loop()
         loop.run_until_complete(server.serve(sockets=self.sockets))
+        # Exit with status 3 when worker starts failed, so Gunicorn
+        # can shut it down to avoid infinite start/stop cycles.
+        # See: https://github.com/encode/uvicorn/issues/1066
+        if not server.started:
+            sys.exit(3)
```

Comment:
It looks like I forgot to send this before :sweat_smile: 

But the suggestion I mentioned is replacing it for:
```suggestion
        if not server.started:
            sys.exit(Arbiter.WORKER_BOOT_ERROR)
```

In the beginning we should also:
```python
from gunicorn.arbiter import Arbiter
```

Reply:
yeah, it's necessary to replace the magic number, I have already pushed a commit to fix it, thanks~